src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gapBetweenCells;
+  padding: $calendarPaddings;
+
+  width: getWidthCalendar(
+    $cellSize, $calendarPaddings, $gapBetweenCells, $thinBorder
+  );
+
+  @extend %thin-border;
+
+  &__day {
+    @include getSquare($cellSize);
+    @extend %flex-center;
+    background-color: #eee;
+    @extend %thin-border;
+    cursor: pointer;
+    transition: transform 0.5s, background-color 0.5s;
+  }
+
+  &__day:hover {
+    background-color: pink;
+    transform: translateY(-20px);
+  }
+
+  @for $index from 1 through 31 {

```

Comment:
Instead of the number 31, you could create a variable, well, it's not fundamental